Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -262,7 +262,41 @@ internal virtual bool SendBytes(byte[] bytes, int channelId = Channels.DefaultRe
                 return false;
             }
 
-            return TransportSend(channelId, bytes);
+            // send to client or server
+            if (Transport.activeTransport.ClientConnected())
+            {
+                return Transport.activeTransport.ClientSend(channelId, bytes);
+            }
+            else if (Transport.activeTransport.ServerActive())
+            {
+                return Transport.activeTransport.ServerSend(connectionId, channelId, bytes);
```

Comment:
You are still using the old ServerSend here.  

we want to move everything to an allocation free SendServer.